atomicwrites/__init__.py

```diff
@@ -19,12 +19,21 @@ def _path_to_unicode(x):
 
 
 if sys.platform != 'win32':
+    def _sync_directory(dst):
+        fd = os.open(os.path.dirname(dst), 0)
+        try:
+            os.fsync(fd)
+        finally:
+            os.close(fd)
+
     def _replace_atomic(src, dst):
         os.rename(src, dst)
+        _sync_directory(dst)
```

Comment:
The manpage of ``rename`` states that the operation is atomic, so this should not be necessary here. But it's still true for ``_move_atomic``.